instructor/ui/instructor.js

```diff
@@ -408,6 +408,20 @@ $(function() {
       window.location.reload();
     });
   });
+
+  /* Create new hrel URL with additionl GET prameters to reopen image chosing window */
```

Comment:
Nitpick, typo, should be `choosing` not chosing.

Reply:
fixed